_benchmark/workloads/index.md

```diff
@@ -0,0 +1,249 @@
+
+### Indices
+
+To create an index, you need to specify the `name` of the index. If you want to add definitions for your index, use the `body` option and point it to the JSON file containing the index definitions.
+
+### Corpora
+
+The `corpora` element requires the name of the index containing the document corpus, for example, `movies`, and a list of parameters that define the document corpora, which includes:
+
+-  `source-file`: The file name to corresponding documents. When using Benchmark locally, documents are contained in a JSON file. When providing a `base_url`, use a compressed file format; `.zip`, `.bz2`, `.gz,` `.tar`, `.tar.gz`, `.tgz` or `.tar.bz2`. The compressed file must contain one JSON file with the name. 
+-  `document-count`  The number of documents in the `source-file` that determines which client indexes correlate to which part of the document corpus (each N client get on N-th of the document corpus). When using a source that contains a document with a parent-child relationship, specify the number of parent documents. 
+- `uncompressed-bytes` The size in bytes of the source file after decompression, used to show you much disk space the decompressed source file needs. You can also indicate the number of `compressed-bytes`, the size of the source file before decompression, which can help you assess the time it'll take for the workload to ingest the documents
+
+### Operations
+
+The `operations` element lists the API OpenSearch operations performed by the workload For example, you can set an operation to `create-index`, which creates an index of Benchmarked data after the workload completes. Operations are usually listed inside of `schedule`.
+
+### Schedule
+
+The `schedule` element contains a list of actions and  operations run by the workload. Operations run according to the order in which they appear in the `schedule`. The following example illustrates a `schedule` with multiple operations, each defined by their `operation-type`. 
+
+```json
+  "schedule": [
+    {
+      "operation": {
+        "operation-type": "create-index"
+      }
+    },
+    {
+      "operation": {
+        "operation-type": "cluster-health",
+        "request-params": {
+          "wait_for_status": "green"
+        },
+        "retry-until-success": true
+      }
+    },
+    {
+      "operation": {
+        "operation-type": "bulk",
+        "bulk-size": 5000
+      },
+      "warmup-time-period": 120,
+      "clients": 8
+    },
+    {
+      "operation": {
+        "name": "query-match-all",
+        "operation-type": "search",
+        "body": {
+          "query": {
+            "match_all": {}
+          }
+        }
+      },
+      "iterations": 1000,
+      "target-throughput": 100
+    }
+  ]
+}
+```
+
+According to this schedule, each action will run in the following order.
+
+1. The `create-index` operation creates an index with benchmarked data.
```

Comment:
`create-index` only creates the index. The index created is empty and does not contain benchmarked data yet (not until the bulk operation occurs where it begins ingesting data into the index)